core/src/main/java/hudson/Functions.java

```diff
@@ -469,6 +469,18 @@ public static Map getSystemProperties() {
         return new TreeMap<Object,Object>(System.getProperties());
     }
 
+    /**
+     * Gets the system property indicated by the specified key.
+     * 
+     * Delegates to {@link SystemProperties#getString(java.lang.String)}.
+     * 
+     * @since TODO
+     */
+    @Restricted(NoExternalUse.class)
```

Comment:
I would say `DoNotUse.class` is enough. We do not want this method to be used internally as well.

Reply:
Changed as requested